ddtrace/contrib/internal/azure_functions/patch.py

```diff
@@ -16,9 +18,10 @@
 
 config._add(
     "azure_functions",
-    {
-        "_default_service": schematize_service_name("azure_functions"),
-    },
+    dict(
+        _default_service=schematize_service_name("azure_functions"),
+        distributed_tracing=asbool(os.getenv("DD_AZURE_FUNCTIONS_DISTRIBUTED_TRACING", default=True)),
```

Comment:
nit: Should these be documented in https://ddtrace.readthedocs.io/en/stable/configuration.html anywhere?

Right now the only want to know about this option is to dig though the code to disable.

Reply:
It is documented, though not explicitly, with `DD_<INTEGRATION>_DISTRIBUTED_TRACING`. This applies to the other undocumented environment variables `DD_BOTTLE_DISTRIBUTED_TRACING`, `DD_CELERY_DISTRIBUTED_TRACING`, `DD_HTTPX_DISTRIBUTED_TRACING`, etc.

Reply:
Good point! Marking this as resolved.